app/assets/scripts/components/documents/single-view/document-body.js

```diff
@@ -925,17 +926,27 @@ export const atbdContentSections = [
         >
           {referencesInUse.length ? (
             <ReferencesList>
-              {referencesInUse.map(({ docIndex, refId }) => {
-                const ref = (document.publication_references || []).find(
-                  (r) => r.id === refId
-                );
-                return (
-                  <li key={refId}>
-                    [{docIndex}]{' '}
-                    {ref ? formatReference(ref) : 'Reference not found'}
-                  </li>
-                );
-              })}
+              {referencesInUse
+                .sort(function (a, b) {
+                  const refA = (document.publication_references || []).find(
+                    (r) => r.id === a.refId
+                  );
+                  const refB = (document.publication_references || []).find(
+                    (r) => r.id === b.refId
+                  );
+
+                  return sortReferences(refA, refB);
+                })
+                .map(({ refId }) => {
+                  const ref = (document.publication_references || []).find(
+                    (r) => r.id === refId
+                  );
+                  return (
+                    <li key={refId}>
+                      {ref ? formatReference(ref) : 'Reference not found'}
+                    </li>
+                  );
+                })}
```

Comment:
We should memoize this since, it consist a `sort` and also a `find` inside a `map`